src/Automation/Automation/help/Register-AzAutomationDscNode.md

```diff
@@ -24,6 +24,12 @@ Register-AzAutomationDscNode -AzureVMName <String> [-NodeConfigurationName <Stri
 ## DESCRIPTION
 The **Register-AzAutomationDscNode** cmdlet registers an Azure virtual machine as an APS Desired State Configuration (DSC) node in an Azure Automation account.
 
+If you need to register a node to an automation account in a different subscription,
```

Comment:
Why is this the case? ARM supports this fnctionality, and cmdlets could support it if they chose to.

Reply:
We will not have capacity for this work during Vb.